knowledge-base/hosting-with-medium-trust-request-for-permission-failed.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Hosting the Reporting REST Service in Medium Trust environment leads to a permission error.
+description: A permissions-related error is thrown in the report viewer when trying the Reporting REST service is hosted in Medium Trust environment.
```

Comment:
If we know the exact error, you can change that to A "exact error" is thrown in the report viewer when trying the Reporting REST service is hosted in Medium Trust environment."